ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/address-locking.md

```diff
@@ -101,9 +101,9 @@ address locking
 
 | Interface | Description | Mode | VLANs | Native VLAN | Trunk Group | Channel-Group |
 | --------- | ----------- | ---- | ----- | ----------- | ----------- | ------------- |
-| Ethernet1 |  Address Locking Interface Testing 1 | access | - | - | - | - |
-| Ethernet2 |  Address Locking Interface Testing 2 | access | - | - | - | - |
-| Ethernet3 |  Address Locking Interface Testing 3 | access | - | - | - | - |
+| Ethernet1 |  Address Locking Interface Testing 1 | - | - | - | - | - |
```

Comment:
Two spaces in front of the description. Since we will be modifying everything, I think we should fix that here.

Reply:
fixed